app/models/solr_document.rb

```diff
@@ -30,4 +30,9 @@ class SolrDocument
   # Recommendation: Use field names from Dublin Core
   use_extension( Blacklight::Solr::Document::DublinCore)    
 
+  def photo_path
```

Comment:
I'd put `BuildingOregon::FileDistributor.new(self.id)` in a private method so you can just do
```
def photo_path
  file_distributor.path
end
```